browser/BUILD.gn

```diff
@@ -175,6 +175,7 @@ source_set("browser_process") {
     "//content/public/browser",
     "//content/public/common",
     "//extensions/buildflags",
+    "//extensions/common:common_constants",
```

Comment:
this should be inside `enable_extensions` guard

Reply:
Addressed by 97c915735c35d17a919dfa8c0a26a0f238d431fe